reliquery/storage.py

```diff
@@ -962,6 +964,159 @@ def get_all_relic_data(self) -> List[Dict]:
         return relic_data
 
 
+class GoogleCloudStorage(Storage):
+    def __init__(
+        self,
+        prefix: str,
+        name: str,
+        token_file: str,
+    ):
+        self.prefix = prefix
+        self.name = name
+        self.token_file = token_file
+
+        self.storage_client = storage.Client.from_service_account_json(self.token_file)
+        # List buckets
+        buckets_list = list(self.storage_client.list_buckets())
+        prefix_bucket = False
+        for bucket in buckets_list:
+            if bucket.id == prefix:
+                self.bucket_id = bucket.id
+                prefix_bucket = True
+        # create prefix bucket if not found
```

Comment:
Let's avoid creating if not found. I'd rather have this error than create the bucket for the user.